python/fbprophet/diagnostics.py

```diff
@@ -186,7 +186,7 @@ def map(self, func, *iterables):
     else:
         predicts = [
             single_cutoff_forecast(df, model, cutoff, horizon, predict_columns)
-            for cutoff in tqdm(cutoffs)
+            for cutoff in (tqdm(cutoffs) if disable_tqdm==False else cutoffs)
```

Comment:
I might hit a very sensitive subject here: You compare booleans, which is never a really good idea. It maybe a good idea to not use `==False`, as recommended by PEP8 standards (see link below). My suggestion is:

```suggestion
for cutoff in (tqdm(cutoffs) if not disable_tqdm else cutoffs)
```

See [PEP8 - Programming Recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) and this [StackOverflow Thread from 2011](https://stackoverflow.com/questions/4591125/is-it-safe-to-replace-with-is-to-compare-boolean-values).

Best
MJT

Reply:
Fair point, I'll commit your suggestion.
Thank you.